pkg/e2e/verify/strict_transport_security.go

```diff
@@ -37,9 +37,10 @@ var _ = ginkgo.Describe(hstsTestName, label.E2E, func() {
 
 func hstsManagedRoutes(ctx context.Context, h *helper.H, namespace string) (bool, error) {
 	route, err := h.Route().RouteV1().Routes(namespace).List(ctx, metav1.ListOptions{})
+	oneYearInSeconds := "31536000"
```

Comment:
I'm not fully sure what we are trying to achieve here.

Reply:
Only making the hardcoding simpler to read/maintain.